### PR TITLE
fix: Don't set public url value to internal url settings.

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
### What does this PR do?
Don't set public url value to internal url settings.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/18643

### PR Checklist
          
[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [x] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [Custom resource definition file is up to date](https://github.com/eclipse-che/che-operator/blob/main/README.md#updating-custom-resource-definition-file)
- [ ] [Nightly OLM bundle is up to date](https://github.com/eclipse-che/che-operator/blob/main/README.md#update-nightly-olm-bundle)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

Signed-off-by: Oleksandr Andriienko <oandriie@redhat.com>

### CODE REVIEW

**Review by tolusha — APPROVED** [review]

**Review by mmorhun — APPROVED** [review]

File: pkg/deploy/server/server_configmap.go [hunk 1]
```diff
@@ -76,6 +77,7 @@ type CheConfigMap struct {
 	DevfileRegistryUrl                     string `json:"CHE_WORKSPACE_DEVFILE__REGISTRY__URL,omitempty"`
 	DevfileRegistryInternalUrl             string `json:"CHE_WORKSPACE_DEVFILE__REGISTRY__INTERNAL__URL,omitempty"`
 	WebSocketEndpointMinor                 string `json:"CHE_WEBSOCKET_ENDPOINT__MINOR"`
+	WebSocketInternalEndpointMinir         string `json:"CHE_WEBSOCKET_INTERNAL_ENDPOINT__MINOR"`
```
Comment (tolusha) [inline pkg/deploy/server/server_configmap.go:80]:
```suggestion
	WebSocketInternalEndpointMinor         string `json:"CHE_WEBSOCKET_INTERNAL_ENDPOINT__MINOR"`
```

**Comment by tolusha** [comment]
devworkspace tests part failed.

File: pkg/deploy/server/server_configmap.go (hunk 1 above)
Comment (AndrienkoAleksandr) [reply pkg/deploy/server/server_configmap.go]: I talked with platform team. They said websocket minor feature was removed one year ago. It's no make sense to leave it in the code, so I removed it.

**Pull request merged**